webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -0,0 +1,24 @@
+def calculate_pauschbetrag(pflegegrad=False, beh_grad=None, merkzeichen_bl=False, merkzeichen_tbl=False, merkzeichen_h=False,
+                           merkzeichen_ag=False, merkzeichen_g=False):
```

Comment:
I think we should talk about naming of the fields across our different PRs. (e.g. I used english names^^)

Reply:
Yes, I assumed that we have to change those.